script/Arcueid/Day 5/05_00_ARC05_9_2.txt

```diff
@@ -12,7 +12,8 @@ Or rather, something more akin to falling.
 [sha:ec152f17a92ded8286eb9b792d3ca1f19305b1b4]{
 -- Page 10021, Offset 10385.
```

Comment:
I just remembered why I thought sliding was better.

It's because everytime someone I know has read this scene they've commented "shouldn't this be sliding?"